tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -528,13 +528,13 @@ static void umode_enter_func(void)
 }
 
 /**
-* @brief Test to check supervisor thread enter one-way to usermode
-*
-* @details A thread running in supervisor mode must have one-way operation
-* ability to drop privileges to user mode.
-*
-* @ingroup kernel_memprotect_tests
-*/
+ * @brief Test to check supervisor thread enter one-way to usermode
```

Comment:
It better to change it like below.
Check if supervisoer thread can enter usermode with one-way operation.

Reply:
@LeiW000  Sorry for reply late. Got it fix, thanks!